pypo crashes during playout when local cache copy of audio file is overwritten due to file size check mismatch

**Describe the bug**

Play-out occasionally stops while files were scheduled in the respective show event. During the silence the On air light isn't lit.

ls_script.log is showing this:

```
2019/03/01 08:03:39 [cue_cut_5774:3] Cueing out...
2019/03/01 08:03:39 [s4:3] Finished with "/var/tmp/airtime/pypo/cache/scheduler/13633.mp3".
2019/03/01 08:03:39 [amplify_5778:3] End of the current overriding.
2019/03/01 08:03:39 [cross_5781:3] No next track ready yet.
2019/03/01 08:03:39 [decoder:3] Method "MAD" accepted "/var/tmp/airtime/pypo/cache/scheduler/13645.mp3".
2019/03/01 08:03:39 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/03/01 08:03:39 [s4:3] Prepared "/var/tmp/airtime/pypo/cache/scheduler/13645.mp3" (RID 0).
2019/03/01 08:03:39 [amplify_5778:3] Overriding amplification: 0.550808.
2019/03/01 08:03:39 [lang:3] timeout --signal=KILL 45 pyponotify --media-id=38074 &
```

It seems to happen quite often:
```
2019/02/28 10:03:36 [dummy(dot)1:3] Source failed (no more tracks) stopping output...
2019/02/28 10:03:36 [dummy(dot)2:3] Source failed (no more tracks) stopping output...
2019/02/28 10:03:36 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/02/28 10:03:36 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 10:03:37 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/02/28 10:59:59 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 10:59:59 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/02/28 12:00:36 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/02/28 13:00:00 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 13:00:00 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/02/28 13:59:55 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 13:59:55 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/02/28 14:59:54 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 14:59:59 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 14:59:59 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/02/28 16:00:00 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 16:00:00 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/02/28 17:03:38 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/02/28 17:03:38 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 17:03:45 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 18:04:00 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 20:04:01 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/02/28 22:04:01 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/03/01 02:03:45 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/03/01 02:03:51 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
2019/03/01 02:03:51 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/03/01 03:03:46 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/03/01 05:03:40 [dummy(dot)4:3] Source failed (no more tracks) stopping output...
2019/03/01 08:03:39 [dummy(dot)3:3] Source failed (no more tracks) stopping output...
```

I'm not sure if this related to #113

**To Reproduce**
Steps to reproduce the behavior:
1. Schedule an event
2. Add MP3 files to the event
3. Wait for the event to play-out the files
4. See error

**Expected behavior**
No silence gaps should occur. LibreTime should immediately play-out the next scheduled file.

**LibreTime version:**
2cda6fa

**Installation method:**
 - OS: Ubuntu
 - OS Version 16.04.5 LTS (Xenial Xerus)
 - Method: ./install` script
 - Details: ./install

**Screenshots**
<none>

**Client (please complete the following information if applicable):**
 - OS: Windows 10
 - Browser Chrome
 - Version latest

**Additional context**
<none>

After updating the erlang and rabbitmq I'm still having the same problem.
 ```
2019-03-20 08:54:45,099 [pypoliqqueue] [INFO ]  waiting 3999.900068s until next scheduled item
2019-03-20 09:02:45,052 [pypofetch] [INFO ]  Queue timeout. Fetching schedule manually
2019-03-20 09:02:45,145 [pypofetch] [ERROR]  end - start length: 3510.0
2019-03-20 09:02:45,146 [pypofetch] [ERROR]  cue_out - cue_in length: 3508.249
2019-03-20 09:02:45,146 [pypofetch] [ERROR]  Two lengths are not equal!!!
2019-03-20 09:02:45,148 [pypofetch] [INFO ]  Loop #3705
```

--- a/python_apps/pypo/pypo/pypofile.py
+++ b/python_apps/pypo/pypo/pypofile.py
@@ -49,10 +49,13 @@ class PypoFile(Thread):
 
         do_copy = False
         if dst_exists:
-            if src_size != dst_size:
-                do_copy = True
-            else:
-                self.logger.debug("file %s already exists in local cache as %s, skipping copying..." % (src, dst))
+            # TODO: Check if the locally cached variant of the file is sane.
+            # This used to be a filesize check that didn't end up working.
+            # Once we have watched folders updated files from them might
+            # become an issue here... This needs proper cache management.
+            # https://github.com/LibreTime/libretime/issues/756#issuecomment-477853018
+            # https://github.com/LibreTime/libretime/pull/845
+            self.logger.debug("file %s already exists in local cache as %s, skipping copying..." % (src, dst))
         else:
             do_copy = True
 

